examples/geoflow/1a63.in

```diff
@@ -2,13 +2,14 @@ read
     mol pqr 1a63.pqr
 end
 elec name comp_solv        # Solvated complex
-    geoflow
+    geoflow-auto
 
     lpbe
     bcfl mdh                # only supported boundary condition
     mol 1
 
-    grid 0.25 0.25 0.25     # grid spacing                    
+    # WARN: grid is not longer supported
+    # grid 0.25 0.25 0.25     # grid spacing                    
```

Comment:
Why is `grid` not supported?

Reply:
The `grid` keyword is not listed in the [geoflow-auto](https://apbs.readthedocs.io/en/latest/using/input/elec/geoflow-auto.html) specification. Is the documentation incorrect? Does `grid` need to be added to the `geoflow-auto` spec?

Reply:
That's odd.  Does the example run successfully?

Reply:
I am adding `grid` into the `geoflow-auto` as keywords. This will allow the current code to run and the parser to still work. I am adding a check_depricated function to WARN the use and point out the keyword that should be changed.